metrics/metricsKey/constants.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2018 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metricsKey
+
+const (
```

Comment:
Are the codes under `knative/pkg/metrics` for stackdriver only? Could you add a doc.go to include the detail description for this package? For example: https://github.com/knative/pkg/blob/master/changeset/doc.go